docs/registry/index.rst

```diff
@@ -154,7 +170,7 @@ interactive data discovery, however, it is usually preferable to use the
 
 And to look for tap resources *in* a specific cone, you would do
 
-.. doctest-remote-data::
+.. doctest-remote-data:: # doctest: +IGNORE_OUTPUT
```

Comment:
This won't work, the #doctest commands should be added at the end of the lines (each one of them) that are affected. But this example already has it.